source/_integrations/event.mqtt.markdown

```diff
@@ -250,4 +250,4 @@ mqtt:
         {% endfor %} }
 ```
 
-{% endraw %}
\ No newline at end of file
+{% endraw %}
```

Comment:
Not sure what changed here

Reply:
No problem. endraw should be there.

Reply:
Yes, I know, but it was there before as well. Could it be that it's a different line ending?

Reply:
Something like that yes. Or a trailing space. Something the lingers missed